TASVideos/Pages/Publications/View.cshtml

```diff
@@ -4,7 +4,9 @@
 	ViewData["Title"] = $"Movie #{Model.Id}";
 }
 @section PageTitle {
-	<h1>@Model.Publication.Title</h1>
+	<div class="container">
+		<h1 style="margin: 0;border-bottom-left-radius: 0;border-bottom-right-radius: 0;width: 100%;">Publication @Model.Publication.Id</h1>
```

Comment:
We have bootstrap classes for some of this stuff.  For the rest, it belongs in the .css file not inline